requirements.txt

```diff
@@ -1,14 +1,17 @@
-fastapi==0.67.0
-fastapi_utils==0.2.1
-gunicorn==20.1.0
-numpy==1.21.1
+numpy==1.20.3
+requests==2.25.1
+pydantic==1.7.4
 psycopg2-binary==2.9.1
+fastapi==0.68.0
+fastapi_utils==0.2.1
 python-dotenv==0.19.0
 
 --find-links https://download.pytorch.org/whl/torch_stable.html
 torch==1.9.0+cpu
 
-transformers==4.9.1
 tweepy==3.10.0
-uvicorn[standard]==0.14.0
+uvicorn[standard]==0.15.0
 SQLAlchemy==1.4.22
+requests_oauthlib==1.3.0
+transformers==4.9.2
+
```

Comment:
Just looking for additional stuff to do, more additions than deletions.  
- Do we need to or should we build a function to delete certain DMs from the inbox?  
- Same goes with Tweets.  
- We could probably go through and put logging at each try & except
- There's a couple spots where I'm worried the app will hard break -- adding error handling would be cool.

Reply:
I will have to check Tweepy to see if they have the functionality for deleting DMs or tweets for that matter. If Tweepy doesn't we can use the manual oauth to get around that. 

Yes we need more error handling all around, wherever you think it can go I would say do it.

Reply:
Sweet, I'll take a crack at it later today